scripts/cli-test.sh

```diff
@@ -51,7 +51,7 @@ until [ "$($FM_BTC_CLIENT getreceivedbyaddress $PEG_OUT_ADDR 0)" == "0.00000500"
 done
 mine_blocks 10
 RECEIVED=$($FM_BTC_CLIENT getreceivedbyaddress $PEG_OUT_ADDR)
-[[ "$RECEIVED" = "0.00000500" ]]
+[[ "$RECEIVED" == "0.00000500" ]];
```

Comment:
Well, if we're already touching it ... why not a proper error message instead of failing silently? :)